sklearn/ensemble/tests/test_iforest.py

```diff
@@ -295,6 +295,34 @@ def test_score_samples():
                        clf2.score_samples([[2., 2.]]))
 
 
+@pytest.mark.filterwarnings('ignore:default contamination')
+@pytest.mark.filterwarnings('ignore:behaviour="old"')
+def test_iforest_warmstart():
+    """Test iterative addition of iTrees to an iForest """
+
+    rng = check_random_state(0)
+
+    # Generate regular observations
+    X = 0.3 * rng.randn(90, 2)
+    X_reg = np.r_[X + 4, X - 4]
+    # Generate some abnormal observations
+    X_outliers = rng.uniform(low=-2, high=2, size=(20, 2))
+    X_train = np.r_[X_reg, X_outliers]
```

Comment:
Do you really need to have outliers for this test? Wouldn't reusing iris.data work too?

Reply:
you're right, not necessary, changed